Sources/MongoSwift/BSON/BSONValue.swift

```diff
@@ -730,12 +741,12 @@ extension String: BSONValue {
         }
     }
 
-    public init(from iter: DocumentIterator) throws {
+    public static func from(iterator iter: DocumentIterator) throws -> BSONValue {
```

Comment:
I forget if I raised this point elsewhere, but would it make sense to drop the "iterator" label here in favor of `_` since there's only one argument? That would keep the new syntax closer to the old `init(from: ...)` API, since "from" is now the method name.

Reply:
Hm, perhaps. I actually kept this in because now calls look like:
```swift
from(iterator: myIter)
```

Which looked pretty nice to me cause it reads like English ("from the iterator under the variable: `myIter`").

I don't feel very passionate about this so we can change it too, I think `from(myIter)` looks and reads fine as well!

Reply:
I'm fine with either of these

Reply:
@Utagai: Good point that this will lead to self-documenting code.

No objections with leaving as-is.